lib/fizzy/types.hpp

```diff
@@ -218,7 +218,7 @@ struct Code
 // https://webassembly.github.io/spec/core/binary/modules.html#data-section
 struct Data
 {
-    MemIdx memidx;
+    // Memory index is ensured to be 0 in the parser
```

Comment:
This comment probably should be before `struct Data`.

Reply:
This one is left to fix.

Reply:
Why?

Reply:
This way you are documenting `ConstantExpression offset` field.

Reply:
Ok, that makes sense, reworded and moved it.